packages/kit/src/runtime/client/client.js

```diff
@@ -241,8 +241,6 @@ export function create_client({ target, base, trailing_slash }) {
 		// abort if user navigated during update
 		if (token !== current_token) return false;
```

Comment:
i think we can get rid of `current_token` now?

```suggestion
		if (token !== nav_token) return false;
```